### PR TITLE
ショッピングカートの中身表示を修正しました

### PR DESCRIPTION
**Pull request by SekiMoemi**

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 1]
```diff
@@ -56,7 +56,13 @@ public String showOrderConfirm(@AuthenticationPrincipal LoginUser user, Model mo
 	 * @return 注文完了画面
 	 */
 	@GetMapping("/finishedOrder")
-	public String toFinishedOrder() {
+	public String toFinishedOrder(Model model, @AuthenticationPrincipal LoginUser user) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:59]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 2]
```diff
@@ -56,7 +56,13 @@ public String showOrderConfirm(@AuthenticationPrincipal LoginUser user, Model mo
...
+		Order order = showShoppingCartService.showShoppingCart(user.getUser().getId());
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:60]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:60]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 3]
```diff
@@ -56,7 +56,13 @@ public String showOrderConfirm(@AuthenticationPrincipal LoginUser user, Model mo
...
+		Integer size = 0;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:61]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:61]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 4]
```diff
@@ -56,7 +56,13 @@ public String showOrderConfirm(@AuthenticationPrincipal LoginUser user, Model mo
...
+		if (order != null) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:62]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:62]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 5]
```diff
@@ -56,7 +56,13 @@ public String showOrderConfirm(@AuthenticationPrincipal LoginUser user, Model mo
...
+			size = order.getOrderItemList().size();
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be 6.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 6]
```diff
@@ -56,7 +56,13 @@ public String showOrderConfirm(@AuthenticationPrincipal LoginUser user, Model mo
...
+		}
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 7]
```diff
@@ -56,7 +56,13 @@ public String showOrderConfirm(@AuthenticationPrincipal LoginUser user, Model mo
...
+		model.addAttribute("cartSize", size);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java [hunk 8]
```diff
@@ -44,8 +44,12 @@ public String showOrderHistory(@AuthenticationPrincipal LoginUser user, Model mo
 
 		List<Order> orderList = showOrderHistoryService.showOrderByUserId(userId);
 		model.addAttribute("orderList", orderList);
-		System.out.println(orderList);
-		model.addAttribute("cartSize", shoppingCartService.showShoppingCart(userId).getOrderItemList().size());
+		Order order = shoppingCartService.showShoppingCart(userId);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java [hunk 9]
```diff
@@ -44,8 +44,12 @@ public String showOrderHistory(@AuthenticationPrincipal LoginUser user, Model mo
...
+		Integer size = 0;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:48]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:48]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java [hunk 10]
```diff
@@ -44,8 +44,12 @@ public String showOrderHistory(@AuthenticationPrincipal LoginUser user, Model mo
...
+		if (order != null) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:49]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:49]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java [hunk 11]
```diff
@@ -44,8 +44,12 @@ public String showOrderHistory(@AuthenticationPrincipal LoginUser user, Model mo
...
+			size = order.getOrderItemList().size();
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:50]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:50]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be 6.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java [hunk 12]
```diff
@@ -44,8 +44,12 @@ public String showOrderHistory(@AuthenticationPrincipal LoginUser user, Model mo
...
+		}
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:51]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:51]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java [hunk 13]
```diff
@@ -44,8 +44,12 @@ public String showOrderHistory(@AuthenticationPrincipal LoginUser user, Model mo
...
+		model.addAttribute("cartSize", size);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:52]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderHistoryController.java:52]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

**Review by yamada-kensyu — APPROVED** [review]
承認します

**Pull request merged**